index.html

```diff
@@ -6,43 +6,178 @@
-      </p>
-      <div class="socials">
-        <ul>
-          <li>
-            <a href=""><img src="./assets/github.png" alt="github" /></a>
-          </li>
-          <li>
-            <a href=""> <img src="./assets/linkedin.png" alt="linkedin" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./assets//Vector.png" alt="vector1" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./assets//twitter.png" alt="twitter" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./assets//Vector2.png " alt="vector2" /></a>
-          </li>
-        </ul>
-      </div>
-    </section>
+    <main>
+      <header>
+        <div class="toolbar">
+          <span><a href="">John Doe</a></span>
+          <img src="./assets//menu.png" />
+        </div>
+      </header>
+      <section class="headline">
+        <span class="title">Hey there. <br />I'm Alex</span>
+        <span class="sub-title">I'am a software Developer</span>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="socials">
+          <ul>
+            <li>
+              <a href=""><img src="./assets/github.png" alt="github" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="./assets/linkedin.png" alt="linkedin" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//Vector.png" alt="vector1" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//twitter.png" alt="twitter" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//Vector2.png " alt="vector2" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="works">
+        <div class="title">
+          <span>My Recent Works</span>
+          <hr />
+        </div>
+        <div class="work-container">
+          <div class="hero-work">
+            <img src="./assets//hero-work.png" alt="hero" />
+            <span>Multi-Post Stories</span>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">css</div>
+              </li>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
```

Comment:
- [ ] [REQUIRED] Please replace this `div` with a `button` or an `a` tag. The "See project" button is a link `a` tag or `button` tag, and please ensure you follow the button styling  and active state -> Active, Hover and Pressed (see design). This is essential because it will link your users to the project page in the future milestone of this portfolio project.